Add Grid level 3 demos to webkit.org

jdatapple opened the pull request:
#### 6d2074996e8fc1b9269fae4e1fbef32152399d34
<pre>
Add Grid level 3 demos to webkit.org
<a href="https://bugs.webkit.org/show_bug.cgi?id=272987">https://bugs.webkit.org/show_bug.cgi?id=272987</a>

Reviewed by Tim Nguyen.

* Websites/webkit.org/demos/grid3/home/megamenu.jpg: Added.
* Websites/webkit.org/demos/grid3/home/museum.jpg: Added.
* Websites/webkit.org/demos/grid3/home/newspaper.jpg: Added.
* Websites/webkit.org/demos/grid3/home/photo.jpg: Added.
* Websites/webkit.org/demos/grid3/home/toc.css: Added.
* Websites/webkit.org/demos/grid3/index.html: Added.
* Websites/webkit.org/demos/grid3/megamenu/index.html: Added.
* Websites/webkit.org/demos/grid3/megamenu/styles.css: Added.
* Websites/webkit.org/demos/grid3/museum/images/National_Gallery_of_Art.png: Added.
* Websites/webkit.org/demos/grid3/museum/images/a_vestal_1952.5.99.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/basket_1943.8.6537.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/boy_in_blue_coat_1959.11.5.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/catherine_brower_1956.13.5.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/detail_of_painting_on_altar_panel_1943.8.5992.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/feeding_the_bird_1953.5.63.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/flowers_in_a_vase_1976.26.2.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/hunting_scene_with_a_pond_1970.17.102.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/john_stuart-wortle_2nd_baron_wharncliffe_2007.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/keelmen_heaving_in_coals_by_moonlight_1942.9.86.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/mademoiselle_sicot_1963.10.209.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/miss_van_alen_1956.13.14.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/niagara_2014.79.10.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/painted_chest_1943.8.7517.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/portrait_of_a_man_1943.7.3.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/retablo_1943.8.6831.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/self-portrait_dedicated_to_carriere_1985.64.20.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/side_of_chest_1943.8.7520.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/study_for__army_boots___verso__1996.121.6.b.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/the_house_of_representatives_2014.79.27.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/the_island_and_bridge_of_san_bartolomeo_rome_2001.23.1.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/the_japanese_footbridge_1992.9.1.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/the_trap_1994.59.13.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/view_of_the_mall_in_saint_jamespark_1970.17.132.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/index.html: Added.
* Websites/webkit.org/demos/grid3/museum/museum.js: Added.
* Websites/webkit.org/demos/grid3/museum/styles.css: Added.
* Websites/webkit.org/demos/grid3/newspaper/index.html: Added.
* Websites/webkit.org/demos/grid3/newspaper/styles.css: Added.
* Websites/webkit.org/demos/grid3/photos/images/1552px-Dendrobates_azureus_(Dendrobates_tinctorius)_Edit.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Architecture_ApplePark.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Bee.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Beskid_Wyspowy_-_panorama_z_Jaworzynki.JPG: Added.
* Websites/webkit.org/demos/grid3/photos/images/Birds.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Boat_passing_under_Elgin_Bridge_during_blue_hour_(2023)-L1003785.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Bow_Bridge_panorama_Central_Park_(86746p).jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Brandslange.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Bridge_CitySkyline.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Bridge_Forest.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Canary_Wharf_from_Limehouse_London_June_2016_HDR.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Caterpillar.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Circle_Shapes.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Crow.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Daslook._Allium_ursinum,_zwellende_bloemknop._18-04-2022_(actm.)_04.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Dawn_in_Puerto_Encantado.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Deer_Meadow.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Dog_GoldenMix.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Faucet.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Flower_Meadow.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Gonepteryx_rhamni_-_Citron-1.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Heilbronn_-_Bockingen_-_Friedhof_-_Blick_von_der_Kapelle_zum_Haupteingang_(1).jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Keylong_West_Lahaul_Himachal_Oct22_A7C_03375_panorama.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Lama_glama_Laguna_Colorada_2.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/MarinaBayAppleStore_Singapore_Texture.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Mekong_bank_with_stilt_dwellings_and_clouds_at_golden_hour_in_Don_Det_Laos.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Mountains_AerialShot.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Mow_Cop_Castle,_Sep_2023.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Nevelzwam_(Clitocybe_nebularis)_24-11-2020._(actm.)_01.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/New York_ CentralPark.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/New York_GeometricShapes.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Ocean_Mountains.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Olympic_Stadium_Munich_-_Rows_of_Seats,_April_2019_-04.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Palacio_de_Schonbrunn,_Viena,_Austria,_2020-02-02,_DD_10.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Parque_Eagle_River,_Anchorage,_Alaska,_Estados_Unidos,_2017-09-01,_DD_02.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Raindrops_on_white_cedar.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/RedwoodNationalParkPlant.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Schloss_Sigmaringen_2022.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/ScrubjaywithAcorn.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Sidewalk_Tunnel.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/SnowcapMountains_Pano.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/StackedBooks.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/StreetAlley_Lights.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Sunset_at_Noronhas&apos;_beach.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Taiwan_Building_1.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Taiwan_Building_3.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/USA-2016-Puerto_Rico-San_Juan-Castillo_San_Felipe_del_Morro_10.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Window_View.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Woods_Foggy.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Wren.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Yellowstone_National_Park_(WY,_USA),_Grand_Prismatic_Spring_--_2022_--_2514.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Yellowstone_National_Park_(WY,_USA),_Lower_Geyser_Basin_--_2022_--_2401.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/index.html: Added.
* Websites/webkit.org/demos/grid3/photos/script.js: Added.
* Websites/webkit.org/demos/grid3/photos/styles.css: Added.
* Websites/webkit.org/demos/grid3/universal.css: Added.
* Websites/webkit.org/demos/grid3/universal.js: Added.

Canonical link: <a href="https://commits.webkit.org/277745@main">https://commits.webkit.org/277745@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/0fde8883695e3709ba3b2cb7938bceeb539bbfbf

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [![loading-orange](https://github-production-user-asset-6210df.s3.amazonaws.com/3098702/291015173-08c448be-ac0a-4fd6-92a3-8165057445b7.png) 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/48462 "Build is in progress. Recent messages:OS: Monterey (12.6.8), Xcode: 13.4.1; Skipping applying patch since patch_id isn't provided; Checked out pull request; 554 style errors") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/27674 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/51422 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/51150 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/44527 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/33610 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/25197 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/39622 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/49044 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/47/builds/25352 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/41832 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/41/builds/20739 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/22835 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | [✅ 🛠 wpe-skia](https://ews-build.webkit.org/#/builders/52/builds/6519 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/44777 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/43484 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/53055 "Built successfully") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/44/builds/23509 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/50/builds/19835 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/46932 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/24774 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/8/builds/42030 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/45850 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/25579 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/6898 "Built successfully and passed tests") | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/45/builds/24497 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on Websites/webkit.org/demos/grid3/universal.css:
```diff
@@ -0,0 +1,141 @@
+:root {
+  color-scheme: light dark;
+  background: light-dark(white, #222);
+}
```
nit: inconsistent indentation in the file

jdatapple committed:
Add Grid level 3 demos to webkit.org
https://bugs.webkit.org/show_bug.cgi?id=272987

Reviewed by Tim Nguyen.

* Websites/webkit.org/demos/grid3/home/megamenu.jpg: Added.
* Websites/webkit.org/demos/grid3/home/museum.jpg: Added.
* Websites/webkit.org/demos/grid3/home/newspaper.jpg: Added.
* Websites/webkit.org/demos/grid3/home/photo.jpg: Added.
* Websites/webkit.org/demos/grid3/home/toc.css: Added.
* Websites/webkit.org/demos/grid3/index.html: Added.
* Websites/webkit.org/demos/grid3/megamenu/index.html: Added.
* Websites/webkit.org/demos/grid3/megamenu/styles.css: Added.
* Websites/webkit.org/demos/grid3/museum/images/National_Gallery_of_Art.png: Added.
* Websites/webkit.org/demos/grid3/museum/images/a_vestal_1952.5.99.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/basket_1943.8.6537.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/boy_in_blue_coat_1959.11.5.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/catherine_brower_1956.13.5.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/detail_of_painting_on_altar_panel_1943.8.5992.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/feeding_the_bird_1953.5.63.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/flowers_in_a_vase_1976.26.2.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/hunting_scene_with_a_pond_1970.17.102.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/john_stuart-wortle_2nd_baron_wharncliffe_2007.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/keelmen_heaving_in_coals_by_moonlight_1942.9.86.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/mademoiselle_sicot_1963.10.209.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/miss_van_alen_1956.13.14.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/niagara_2014.79.10.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/painted_chest_1943.8.7517.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/portrait_of_a_man_1943.7.3.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/retablo_1943.8.6831.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/self-portrait_dedicated_to_carriere_1985.64.20.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/side_of_chest_1943.8.7520.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/study_for__army_boots___verso__1996.121.6.b.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/the_house_of_representatives_2014.79.27.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/the_island_and_bridge_of_san_bartolomeo_rome_2001.23.1.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/the_japanese_footbridge_1992.9.1.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/the_trap_1994.59.13.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/images/view_of_the_mall_in_saint_jamespark_1970.17.132.jpg: Added.
* Websites/webkit.org/demos/grid3/museum/index.html: Added.
* Websites/webkit.org/demos/grid3/museum/museum.js: Added.
* Websites/webkit.org/demos/grid3/museum/styles.css: Added.
* Websites/webkit.org/demos/grid3/newspaper/index.html: Added.
* Websites/webkit.org/demos/grid3/newspaper/styles.css: Added.
* Websites/webkit.org/demos/grid3/photos/images/1552px-Dendrobates_azureus_(Dendrobates_tinctorius)_Edit.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Architecture_ApplePark.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Bee.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Beskid_Wyspowy_-_panorama_z_Jaworzynki.JPG: Added.
* Websites/webkit.org/demos/grid3/photos/images/Birds.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Boat_passing_under_Elgin_Bridge_during_blue_hour_(2023)-L1003785.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Bow_Bridge_panorama_Central_Park_(86746p).jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Brandslange.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Bridge_CitySkyline.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Bridge_Forest.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Canary_Wharf_from_Limehouse_London_June_2016_HDR.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Caterpillar.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Circle_Shapes.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Crow.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Daslook._Allium_ursinum,_zwellende_bloemknop._18-04-2022_(actm.)_04.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Dawn_in_Puerto_Encantado.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Deer_Meadow.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Dog_GoldenMix.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Faucet.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Flower_Meadow.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Gonepteryx_rhamni_-_Citron-1.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Heilbronn_-_Bockingen_-_Friedhof_-_Blick_von_der_Kapelle_zum_Haupteingang_(1).jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Keylong_West_Lahaul_Himachal_Oct22_A7C_03375_panorama.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Lama_glama_Laguna_Colorada_2.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/MarinaBayAppleStore_Singapore_Texture.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Mekong_bank_with_stilt_dwellings_and_clouds_at_golden_hour_in_Don_Det_Laos.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Mountains_AerialShot.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Mow_Cop_Castle,_Sep_2023.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Nevelzwam_(Clitocybe_nebularis)_24-11-2020._(actm.)_01.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/New York_ CentralPark.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/New York_GeometricShapes.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Ocean_Mountains.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Olympic_Stadium_Munich_-_Rows_of_Seats,_April_2019_-04.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Palacio_de_Schonbrunn,_Viena,_Austria,_2020-02-02,_DD_10.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Parque_Eagle_River,_Anchorage,_Alaska,_Estados_Unidos,_2017-09-01,_DD_02.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Raindrops_on_white_cedar.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/RedwoodNationalParkPlant.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Schloss_Sigmaringen_2022.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/ScrubjaywithAcorn.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Sidewalk_Tunnel.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/SnowcapMountains_Pano.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/StackedBooks.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/StreetAlley_Lights.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Sunset_at_Noronhas'_beach.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Taiwan_Building_1.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Taiwan_Building_3.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/USA-2016-Puerto_Rico-San_Juan-Castillo_San_Felipe_del_Morro_10.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Window_View.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Woods_Foggy.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Wren.jpeg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Yellowstone_National_Park_(WY,_USA),_Grand_Prismatic_Spring_--_2022_--_2514.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/images/Yellowstone_National_Park_(WY,_USA),_Lower_Geyser_Basin_--_2022_--_2401.jpg: Added.
* Websites/webkit.org/demos/grid3/photos/index.html: Added.
* Websites/webkit.org/demos/grid3/photos/script.js: Added.
* Websites/webkit.org/demos/grid3/photos/styles.css: Added.
* Websites/webkit.org/demos/grid3/universal.css: Added.
* Websites/webkit.org/demos/grid3/universal.js: Added.

Canonical link: https://commits.webkit.org/277745@main
